testgen/testdata_gen.py

```diff
@@ -402,6 +402,10 @@ def mapFmtSkeletonToECMA402(options):
     if o != 'scale/0.5' and o != 'decimal-always':
       option_detail = ecma402_map[o]
       options_dict = options_dict | option_detail
+    if o[0:5] == "scale":
+        options_dict = options_dict | {"conformance_scale": o[6:]}
+    if o == "decimal-always":
+        options_dict = options_dict | {"conformance_decimal_always": True}
```

Comment:
Question: shouldn't this be in camel case?

Reply:
Fixed!